core/controllers/collection_editor.py

```diff
@@ -154,6 +140,43 @@ def _require_valid_version(self, version_from_payload, collection_version):
                 'which is too old. Please reload the page and try again.'
                 % (collection_version, version_from_payload))
 
+    # Require editor handles all edit permissions for a collection
+    @require_editor
+    def get(self, collection_id):
+        """Populates the data on the individual collection page."""
+
+        collection = (
+            collection_services.get_collection_by_id(collection_id))
+
+        try:
+            # Try to retrieve collection
+            collection_dict = (
+                summary_services.get_learner_collection_dict_by_id(
+                    collection_id, self.user_id,
+                    allow_invalid_explorations=True))
+        except Exception as e:
+            raise self.PageNotFoundException(e)
+
+        self.values.update({
```

Comment:
All these values don't seem right to me; shouldn't this controller only be passing a collection dict to the frontend, just like it was before?